management/client.go

```diff
@@ -232,14 +232,160 @@ type OIDCBackchannelLogout struct {
 
 // ClientAddons defines the `addons` settings for a Client.
 type ClientAddons struct {
+	AWS                  *AWSClientAddon                  `json:"aws,omitempty"`
+	AzureBlob            *AzureBlobClientAddon            `json:"azure_blob,omitempty"`
+	AzureSB              *AzureSBClientAddon              `json:"azure_sb,omitempty"`
+	Box                  *BoxClientAddon                  `json:"box,omitempty"`
+	CloudBees            *CloudBeesClientAddon            `json:"cloudbees,omitempty"`
+	Concur               *ConcurClientAddon               `json:"concur,omitempty"`
+	Dropbox              *DropboxClientAddon              `json:"dropbox,omitempty"`
+	EchoSign             *EchoSignClientAddon             `json:"echosign,omitempty"`
+	Egnyte               *EgnyteClientAddon               `json:"egnyte,omitempty"`
+	Firebase             *FirebaseClientAddon             `json:"firebase,omitempty"`
+	Layer                *LayerClientAddon                `json:"layer,omitempty"`
+	MSCRM                *MSCRMClientAddon                `json:"mscrm,omitempty"`
+	NewRelic             *NewRelicClientAddon             `json:"newrelic,omitempty"`
+	Office365            *Office365ClientAddon            `json:"office365,omitempty"`
+	RMS                  *RMSClientAddon                  `json:"rms,omitempty"`
+	Salesforce           *SalesforceClientAddon           `json:"salesforce,omitempty"`
+	SalesforceAPI        *SalesforceAPIClientAddon        `json:"salesforce_api,omitempty"`
+	SalesforceSandboxAPI *SalesforceSandboxAPIClientAddon `json:"salesforce_sandbox_api,omitempty"`
+	SAML2                *SAML2ClientAddon                `json:"samlp,omitempty"`
+	SAPAPI               *SAPAPIClientAddon               `json:"sap_api,omitempty"`
+	Sharepoint           *SharepointClientAddon           `json:"sharepoint,omitempty"`
```

Comment:
Should this one be `SharePoint`? Also `SharePointClientAddon`.

Reply:
Yup! I knew I should googled that one to check the casing.